framework/src/base/Moose.C

```diff
@@ -1193,3 +1193,5 @@ bool _deprecated_is_error = false;
 bool _throw_on_error = false;
 
 } // namespace Moose
+
+OStreamProxy & _console = Moose::out;
```

Comment:
I need to research the portability of this more.  Can you really create a *static* reference to what is effectively `std::cout`?  The initialization order of static variables is undefined, so at the time this reference is created, `std::cout` might not exist?